src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -91,6 +91,15 @@ const ProfileSelector = ({ input }: ProfileSelectorProps) => {
       change('kernel', data.kernel);
     }
     if (data?.services?.enabled) {
+      if (data.services.enabled.includes('firewalld')) {
```

Comment:
This is a tad bit ugly (but also temporary). Any suggestions on how to handle this better, or is this okay for now?

Reply:
It can get removed once https://github.com/osbuild/image-builder/pull/1110 lands

Reply:
https://github.com/osbuild/image-builder/pull/1110 seems close to landing. If it gets merged today/tomorrow then I'd say let's wait for it and remove the temporary part right away so we don't forget to remove it :upside_down_face: What do you think?

Reply:
So https://github.com/osbuild/image-builder/pull/1110 got merged, when is this planned to go in?

Reply:
I'll push the changes for this today